LICENSE

```diff
@@ -1,6 +1,9 @@
 The MIT License
 
-Copyright (c) 2017-2020 Denis Fäcke
+Copyright (c) 2017-2024 The Fastify Team
```

Comment:
```suggestion
Copyright (c) 2021-2024 The Fastify Team
```

Reply:
This is the year of the fork

Reply:
It is not even what matteo requested.
He wrote to add another line (below of Denis Fäckes Line)  

```suggestion
Copyright (c) 2017-2020 Denis Fäcke
Copyright (c) 2021-2024 The Fastify Team
```